### ISSUE TITLE
Improve documentation for upgrading mautic for CLI clear instructions how to do it

### ISSUE THREAD

**Issue by dynamiccontent** [mautic/mautic-documentation#197]
Hello,

As a new user of Mautic I am always looking at the official documentation for how to do upgrading properly on a project. Your documentation is lacking official recommended steps to how to perform upgrade properly.

I have been searching on other unofficial sites to see upgrade tips and it seems they cover this topic better then official documentation.
For example this guide https://mauteam.org/mautic/mautic-admins/mautic-update-how-to-dont-click-that-button/ will show the steps that I think are OK to do the upgrade.

Can we modify the documentation so it will contain official steps. This would be added here https://docs.mautic.org/en/mautic-3-upgrade/getting-started#choose-your-update-method-web-interface-or-cli

For CLI method I would recommend documentation like this as I have done my upgrade:

1. Verify the health of your current version (where /var/www/mautic/ is location of your mautic installation):
```
php /var/www/mautic/bin/console doctrine:migration:migrate
```
This will check if there are any pending database migrations and if there are, it will perform the migration.

![upgrade1](https://user-images.githubusercontent.com/56940608/116307783-0c241380-a7a7-11eb-9870-c8f68e7b7408.png)

2. Now we’re gonna make sure there aren’t any differences between your database schema and how it is supposed to be:
```
php /var/www/mautic/bin/console doctrine:schema:update --force
```
This will check if your schema is up to date and if not it will perform the required modifications.

![upgrade2](https://user-images.githubusercontent.com/56940608/116308183-80f74d80-a7a7-11eb-842b-9ce084eabe91.png)

3. Finally, clear the Mautic cache:
```
php /var/www/mautic/bin/console cache:clear
```
![upgrade3](https://user-images.githubusercontent.com/56940608/116308532-e3e8e480-a7a7-11eb-9cad-b40e9671d28b.png)

4: Download the Mautic update package YOU have chosen. Check https://github.com/mautic/mautic/releases and select the update package for example x.x.x-update.zip

Before you download make sure you are in the location you want to save your download
you can check your current location with command
```
pwd
```
my current location is /var/www/
![upgrade4](https://user-images.githubusercontent.com/56940608/116308769-2f02f780-a7a8-11eb-8fe3-3267d4b7a5b3.png)

You can use command below to download the package
```
wget https://github.com/mautic/mautic/releases/download/x.x.x/x.x.x-update.zip
```
![upgrade5](https://user-images.githubusercontent.com/56940608/116309264-da13b100-a7a8-11eb-9876-360932a3588b.png)

5. Now we can perform the upgrade with the command below (where /var/www/x.x.x-update.zip is the location of your upgrade file)
```
php /var/www/mautic/bin/console mautic:update:apply --update-package=/var/www/x.x.x-update.zip
```
![upgrade6](https://user-images.githubusercontent.com/56940608/116309742-6f16aa00-a7a9-11eb-8fb0-a4c47cc1761e.png)

6. finishing the upgrade with command
```
php /var/www/mautic/bin/console mautic:update:apply --finish
```
![upgrade7](https://user-images.githubusercontent.com/56940608/116309890-a08f7580-a7a9-11eb-9260-d10c579ba239.png)

7. To complete run this three commands the same as you did before starting the upgrade.
```
php /var/www/mautic/bin/console doctrine:migration:migrate
php /var/www/mautic/bin/console doctrine:schema:update --force
php /var/www/mautic/bin/console cache:clear
```
![upgrade8](https://user-images.githubusercontent.com/56940608/116310304-1dbaea80-a7aa-11eb-9df3-64f5858241ca.png)

Well done. You successfully upgraded your Mautic. 

Make sure to check Current Error Log on your Mautic after upgrading.
![upgrade9](https://user-images.githubusercontent.com/56940608/116310648-9457e800-a7aa-11eb-96c0-c5221d0a5448.png)

This kind of instructions would help me a lot when upgrading because right now I don't know if I did the upgrade optimally. Also a note if jumping between version is supported so if I missed a few releases and I want to upgrade to latest what would be the officially supported upgrade path?

Any advice or discussion about this would be valuable.

Thank you!

**Comment by dynamiccontent** [comment]
Let me just reference this topic https://github.com/mautic/mautic-documentation/issues/113 and people in the topic to get some discussion going:
@StoppingBuck @RCheesley @ZachHoffman208
Thank you.

**Comment by RCheesley** [comment]
Hey there! 

At the bottom of every page in the documentation is a link to edit the page - feel free to go ahead and then make the pull request for the team to update!

Shout if you need any help with that - please join #t-education on Slack (https://mautic.org/slack) if you would like to help with the docs - we definitely agree they need to be improved!

**Comment by dynamiccontent** [comment]
@RCheesley Before I do that I would like some confirmation that this upgrade steps are correct and that this is optimal way to do the upgrade?

**Comment by RCheesley** [comment]
Doing an update should be:

```
bin/console m:u:f
bin/console m:u:a
bin/console c:c
```

There should not be any need for all the other steps, indeed a Mautic 3.x instance that has been upgraded may well show a bunch of migrations that are no longer relevant. You would only do those steps if something was broken with a past update, generally speaking.

**Comment by dynamiccontent** [comment]
@RCheesley that is interesting as I was upgrading from Mautic 3.3.2 fresh install to Mautic 3.3.3 and as you can see in the images it showed doing all that so I guess it does not hurt to do the extra steps to make sure the install is not broken?

**Comment by RCheesley** [comment]
Hey there, I have started working on a PR here: https://github.com/mautic/mautic-documentation/pull/229 feel free to review/add comments.
